Admin.js

```diff
@@ -11,7 +11,7 @@
 
```

Comment:
- eslint: Line 226 exceeds the maximum line length of 80.
- eslint: Strings must use doublequote.
- eslint: Expected '===' and instead saw '=='.
- eslint: No magic number: 0.
- eslint: Block must be padded by blank lines.